apps/agreement/test/agreement/agreement_new_action.js

```diff
@@ -156,6 +226,9 @@ contract('Agreement', ([_, owner, submitter, someone]) => {
 
         context('when the app is unregistered', () => {
           beforeEach('mark as unregistered', async () => {
+            await deployer.arbitrator.setModule(deployer.TRANSACTION_FEES_MODULE, deployer.transactionFeesOracle.address)
+            const { collateralToken } = disputable
+            await deployer.transactionFeesOracle.setFee('0x', collateralToken.address, 0)
```

Comment:
Why do you need to set the module here? Isn't it already being set in the deployer?